Description: Remove unused import from cinnamon-settings
 cinnamon-settings imports in its python  scripts Gst and GtkClutter which
 it does nothing with. So remove them to avoid crash in case of uninstalled
 typelib files.
Author: Nicolas Bourdaud <nicolas.bourdaud@gmail.com>
Origin: Debian
Bug-Debian: http://bugs.debian.org/704919
Forwarded: https://github.com/linuxmint/Cinnamon/pull/1895
Last-Update: 2013-04-21
--- a/files/usr/lib/cinnamon-settings/cinnamon-settings.py
+++ b/files/usr/lib/cinnamon-settings/cinnamon-settings.py
@@ -8,7 +8,7 @@
     import os
     import glob
     import gettext
-    from gi.repository import Gio, Gtk, GObject, GdkPixbuf, GtkClutter, Gst
+    from gi.repository import Gio, Gtk, GObject, GdkPixbuf
     import SettingsWidgets
     import capi
 # Standard setting pages... this can be expanded to include applet dirs maybe?
